include/odp/api/spec/traffic_mngr.h

```diff
@@ -189,6 +189,47 @@ extern "C" {
  * tree/hierarchy of nodes.
  */
 
+/**
+ * TM node stats capability
+ */
+typedef struct odp_tm_node_stats_capability_t {
+	/** Supported counters */
+	union {
+		/** Statistics counters supported in bit field struct */
+		struct {
+			/** @see odp_tm_node_stats_t::dropped_pkts */
+			uint64_t dropped_pkts;
```

Comment:
It seems these capability fields were supposed to be bit fields, but now they are not defined as such.

Reply:
I missed it mistakenly. Will fix it in v2.